### ISSUE TITLE
Unable to build, seems to be sigchain issue

### ISSUE THREAD

**Issue by SkYNewZ** [kryptco/kr#254]
Os: Manjaro
Kernel : 4.20

```
[...]
/tmp/kr/src/src/github.com/kryptco/kr/sigchain/cross/linux-x86_64/share/doc/openssl/html/man5/x509v3_config.html
make[2]: Leaving directory '/tmp/kr/src/src/github.com/kryptco/kr/sigchain/openssl'
cargo web --version || (echo 'Please install cargo-web with `cargo install cargo-web`and make sure $HOME/.cargo/bin is on your $PATH' ; exit 1;)
cargo-web 0.6.10
# use rsync so that file modifed times are only updated when the contents change
cd dashboard_yew && cargo web deploy --release --target=wasm32-unknown-emscripten && rsync --checksum --delete -r ../target/deploy/* ../target/deploy-final
error: error: failed to parse manifest at `/tmp/kr/src/src/github.com/kryptco/kr/sigchain/Cargo.toml`

Caused by:
  virtual manifests do not specify [target]

make[1]: *** [Makefile:25: libsigchain-with-dashboard] Error 101
make[1]: Leaving directory '/tmp/kr/src/src/github.com/kryptco/kr/sigchain'
make: *** [Makefile:50: all] Error 2
==> ERROR: A failure occurred in build().
    Aborting...
```

**Comment by SkYNewZ** [comment]
By doing `cd dashboard_yew && cargo web deploy --release --target=wasm32-unknown-emscripten` manually (https://github.com/kryptco/sigchain/blob/master/Makefile#L25), I get
```
error: error: failed to parse manifest at `/home/quentin/go/src/github.com/kryptco/kr/sigchain/Cargo.toml`

Caused by:
  virtual manifests do not specify [target]
```

**Comment by witwender** [comment]
I'm getting the exact same issue on Manjaro Gnome with 4.20.

**Comment by SkYNewZ** [comment]
YES I'M NOT ALONE :satisfied:

**Comment by oldsj** [comment]
I have this issue as well. You can get around it for now by removing the android target config in sigchain/Cargo.toml:
`sed -i.bak -e '8,11d' sigchain/Cargo.toml`

**Comment by SkYNewZ** [comment]
I'm on Android, if I do that, can I still use Kryptco?

**Comment by witwender** [comment]
Ok, noob question time. How do you build like that? Using Pamac and Yay, Kr package seems to be cloned again at build time and I cannot seem to be able to build with that option.

**Comment by SkYNewZ** [comment]
You can read this for more understanding : https://wiki.archlinux.org/index.php/Makepkg#Usage

When you use a AUR manager, it do the job for you but you can do yourself by doing :
- `git clone` the package (link available on AUR package page)
- go into
- do some stuff if you want to customize the package (example here fix the build)
- and finally install like yay can do : `makepkg -i`

**Comment by Arsur** [comment]
This error still persist. Even with this fix i wasnt able to build it under arch. Some other advise?

**Comment by marcus-grant** [comment]
Why does this sigchain issue disrupt the installation? I'll try to make a fix to the AUR package to get around this, but it seems like it's an unsustainable fix because this config file could just be modified in the future and break my change to the pkg. Should I attempt this fix to the AUR package or will this team fix either the build process or is this a problem with cargo-web?

**Comment by viggy96** [comment]
This issue still occurs on Manjaro. 

```
error: error: failed to parse manifest at `/var/tmp/pamac-build-vignesh/kr/src/src/github.com/kryptco/kr/sigchain/Cargo.toml`

Caused by:
  virtual manifests do not specify [target]

make[1]: *** [Makefile:25: libsigchain-with-dashboard] Error 101
make[1]: Leaving directory '/var/tmp/pamac-build-vignesh/kr/src/src/github.com/kryptco/kr/sigchain'
make: *** [Makefile:50: all] Error 2
==> ERROR: A failure occurred in build().
    Aborting...
```

**Comment by Fuzen-py** [comment]
Fixed with 05cd45c05684de51a03c9e3b7bf3bc32616051c7 but it looks like aur doesn't have 2.4.15  yet
